src/helpers.ts

```diff
@@ -478,3 +498,41 @@ export function updateProjectIfMinterConfigIsActive(
     project.save();
   }
 }
+
+/**
+ * Calculates the total auction time for an exponential Dutch auction.
+ * @param startPrice The starting price of the auction.
+ * @param basePrice The base price of the auction.
+ * @param halfLifeSeconds The half life of the auction.
+ * @returns The total auction time in seconds.
+ **/
+export function getTotalDAExpAuctionTime(
+  startPrice: BigInt,
+  basePrice: BigInt,
+  halfLifeSeconds: BigInt
```

Comment:
It seems like it is probably worthwhile to add some smol unit tests for this helper specifically?

Reply:
A single auction is checked via independently written typescript calculation in e2e tests, but agree here and added a couple of spot checks for this function against current prod hasura values in [55bb4ff](https://github.com/ArtBlocks/artblocks-subgraph/pull/241/commits/55bb4ff5fb09a496df4879ed4562b3302505ab31)